app/assets/javascripts/comments.js

```diff
@@ -1,6 +1,7 @@
 $(document).ready(function() {
-  $('.toggle-reply').on('click', function(e) {
+  $(document).on('click', '.toggle-reply', function(e) {
     e.preventDefault();
-    $(e.target).parents("ul").next('.comment-reply').toggleClass('hidden');
+    commentID = $(e.target).data('parent-comment-id');
+    $(e.target).parents("ul").next("#reply-form-"+commentID).toggleClass('hidden');
```

Comment:
`$().parents().next()` is much more fragile if the HTML changes, which it probably will. I also think hunting through the DOM for an ID is pretty fast, considering there should be only one of them.